src/frontend/packages/lib_components/src/common/SchedulingFields/index.tsx

```diff
@@ -219,43 +211,20 @@ export const SchedulingFields = ({
         margin={margin}
         gap="small"
       >
-        <Box>
-          <FormField
-            label={intl.formatMessage(messages.startingAtDateTextLabel)}
-            htmlFor="starting_at_date"
-            margin="none"
-            background={startingAtError ? 'status-error-off' : 'white'}
-            // Fix of the calendar icon still clickable when component disabled (see below)
-            style={{ pointerEvents: disabled ? 'none' : undefined }}
-            height="80%"
-            disabled={disabled}
-          >
-            <DateInput
-              dropProps={{
-                align: { top: 'bottom', left: 'left' },
-                style: {
-                  borderRadius: '4px',
-                  boxShadow: 'rgb(0 0 0 / 23%) 4px 5px 17px',
-                },
-              }}
-              id="starting_at_date"
-              format={intl.locale === 'fr' ? 'dd/mm/yyyy' : 'yyyy/mm/dd'}
-              value={currentStartingAtDate || undefined}
-              onChange={onStartingAtDateInputChange}
-              calendarProps={{
-                bounds: [
-                  DateTime.local().toISO() as string,
-                  DateTime.local().plus({ years: 1 }).toISO() as string,
-                ],
-              }}
-              // TODO : calendar icon still clickable even when component is disabled
-              // need to open an issue on grommet's github
+        <Box data-testid="starting-at-date-picker">
+          <CunninghamProvider>
```

Comment:
Shouldn't CunninghamProvider be at the top of both lti and standalone apps ?

Reply:
I don't think we should put it on the top, one of the pitfall of the Context is that it rerenders his child three when it updates, so we will rerender the full three when actually we need to rerender only the DatePicker.
Better explanation here: https://blog.logrocket.com/pitfalls-of-overusing-react-context/

Reply:
So we will put the context around a cunningham component everytime we want to use one ?